src/coreclr/tools/Common/TypeSystem/Common/TargetArchitecture.cs

```diff
@@ -9,13 +9,15 @@ namespace Internal.TypeSystem
     /// <summary>
     /// Specifies the target CPU architecture.
     /// </summary>
-    public enum TargetArchitecture
+    /// This is used in serialization for symbols. Please don't reorder or reassing values.
+    /// Deletions and renames are OK, additions are only allowed as appends.
+    public enum TargetArchitecture : uint
     {
-        Unknown,
-        ARM,
-        ARM64,
-        X64,
-        X86,
-        Wasm32,
+        Unknown = 0,
```

Comment:
Enums that are part of the TypeSystem are not considered to be part of the File format, and should not be considered to be stable. Please make separate enums in a file which is explicitly for the file format you are working with.